auth/src/main/java/com/firebase/ui/auth/ErrorCodes.java

```diff
@@ -1,20 +1,49 @@
 package com.firebase.ui.auth;
 
+import android.support.annotation.IntDef;
+
+import com.firebase.ui.auth.data.model.FirebaseUiException;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
 /**
- * Error codes retrieved from {@link IdpResponse#getErrorCode()}.
+ * Error codes for failed sign-in attempts.
  */
 public final class ErrorCodes {
     /**
-     * Sign in failed due to lack of network connection
-     **/
-    public static final int NO_NETWORK = 10;
+     * Valid codes that can be returned from {@link FirebaseUiException#getErrorCode()}.
+     */
+    @IntDef({
+                    UNKNOWN_ERROR,
+                    NO_NETWORK,
+                    PLAY_SERVICES_UPDATE_CANCELLED,
+                    DEVELOPER_ERROR
+            })
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface All {}
```

Comment:
Not really happy with this name, any ideas?

Reply:
Maybe `Code`?

Reply:
SGTM, `ErrorCodes.Anything` feels weird, but yours makes more sense.